From: adam radford <aradford@gmail.com>
Date: Mon, 8 Mar 2010 12:40:36 -0800
Subject: [PATCH] [SCSI] 3ware maintainers update
Git-commit: c4de0cebc512148bee764bfdac7bf49555e41173
References: FATE#311054
Patch-Mainline: 2.6.34

This patch updates the 3ware maintainers in the MAINTAINERS file.

Signed-off-by: Adam Radford <aradford@gmail.com>
Signed-off-by: James Bottomley <James.Bottomley@suse.de>
Signed-off-by: Hannes Reinecke <hare@suse.de>
---
 MAINTAINERS |   15 ++++-----------
 1 files changed, 4 insertions(+), 11 deletions(-)

diff --git a/MAINTAINERS b/MAINTAINERS
index 73ee5ac..b3a20a5 100644
--- a/MAINTAINERS
+++ b/MAINTAINERS
@@ -130,19 +130,12 @@ L:	netdev@vger.kernel.org
 S:	Maintained
 F:	drivers/net/typhoon*
 
-3W-9XXX SATA-RAID CONTROLLER DRIVER
-M:	Adam Radford <linuxraid@amcc.com>
+3WARE SAS/SATA-RAID SCSI DRIVERS (3W-XXXX, 3W-9XXX, 3W-SAS)
+M:	Adam Radford <linuxraid@lsi.com>
 L:	linux-scsi@vger.kernel.org
-W:	http://www.amcc.com
+W:	http://www.lsi.com
 S:	Supported
-F:	drivers/scsi/3w-9xxx*
-
-3W-XXXX ATA-RAID CONTROLLER DRIVER
-M:	Adam Radford <linuxraid@amcc.com>
-L:	linux-scsi@vger.kernel.org
-W:	http://www.amcc.com
-S:	Supported
-F:	drivers/scsi/3w-xxxx*
+F:	drivers/scsi/3w-*
 
 53C700 AND 53C700-66 SCSI DRIVER
 M:	"James E.J. Bottomley" <James.Bottomley@HansenPartnership.com>
-- 
1.6.0.2

